command/commands/team.py

```diff
@@ -2,24 +2,29 @@
 import argparse
 import logging
 import shlex
+from flask import jsonify
 from interface.github import GithubAPIException
 from model.team import Team
+from model.user import User
```

Comment:
```suggestion
from model import Team, User
```

Reply:
There are a few other places where this occurs. Search for string `from model\.`.